Tests/ContractTests/NetworkTests.swift

```diff
@@ -50,16 +50,6 @@ class NetworkTests: XCTestCase {
       XCTAssertEqual(span.attributes.first { $0.key == "net.peer.name" }?.value.stringValue, "localhost")
       XCTAssertEqual(span.attributes.first { $0.key == "net.peer.port" }?.value.intValue, "8181")
       XCTAssertEqual(span.attributes.first { $0.key == "network.connection.type" }?.value.stringValue, "unavailable")
-
-      // Session and user attributes
-      XCTAssertNotNil(span.attributes.first { $0.key == "user.id" }?.value.stringValue)
-      XCTAssertNotNil(span.attributes.first { $0.key == "session.id" }?.value.stringValue)
-      XCTAssertNotNil(span.attributes.first { $0.key == "session.previous_id" }?.value.stringValue)
```

Comment:
question: curious, what was the issue?

Reply:
isse was resolved in this commit - https://github.com/aws-observability/aws-otel-swift/pull/84/commits/2d4ffb25185f63f7a70d244e7f07762a7a3a2e96